ibis/expr/datatypes/tests/test_core.py

```diff
@@ -75,6 +75,23 @@ def test_dtype_from_classes(klass, expected):
     assert dt.dtype(klass) == expected
 
 
+@pytest.mark.parametrize(
+    ("uklass", "klass"),
+    [
+        (dt.UInt64, dt.Int64),
```

Comment:
I would just test all 8 options literally to be totally clear and invite no possible problems:

(dt.UInt8, 0, 0xFF),
(dt.Int8, -32768, 32767),
etc